src/actions/index.js

```diff
@@ -1,8 +1,26 @@
 // we'll need axios
+import axios from "axios";
 
 // we'll need to create 3 different action types here.
 // one for fetching, one for success and one for failure
+export const FETCH_START = "FETCH_START";
+export const FETCH_SUCCESS = "FETCH_SUCCESS";
+export const FETCH_FAILED = "FETCH_FAILED";
 
 // our action creator will be a function that returns a function
 // the url to fetch characters from is `https://swapi.co/api/people/`
 // remember that now we have controll over our thunk-based action creator
+export const getCharacters = () => dispatch => {
+  dispatch({ type: FETCH_START });
+  setTimeout(() => {
+    axios
+    .get("https://swapi.co/api/people")
+    .then(res => {
+      dispatch({ type: FETCH_SUCCESS, payload: res.data.results });
+    })
+    .catch(err => {
+      dispatch({ type: FETCH_FAILED, payload: err });
+    });
+  }, 3000)
```

Comment:
Curious how come there is a timeout here? I'm assuming it's to the loader?

Reply:
Yeah I just put that there to see the loader it isn't necessary